src/dotnet/commands/dotnet-pack/Program.cs

```diff
@@ -29,6 +29,7 @@ public static int Run(string[] args)
             var buildBasePath = app.Option("-b|--build-base-path <OUTPUT_DIR>", "Directory in which to place temporary build outputs", CommandOptionType.SingleValue);
             var configuration = app.Option("-c|--configuration <CONFIGURATION>", "Configuration under which to build", CommandOptionType.SingleValue);
             var versionSuffix = app.Option("--version-suffix <VERSION_SUFFIX>", "Defines what `*` should be replaced with in version field in project.json", CommandOptionType.SingleValue);
+            var serviceable = app.Option("-s|--serviceable", "Set the serviceable flag in the output nuspec", CommandOptionType.NoValue);
```

Comment:
"the output nuspec" seems an odd phrase to an outside user.  The user is calling `dotnet pack` and the output is a .nupkg.